### ISSUE TITLE
how to resolve this

### ISSUE THREAD

**Issue by rinky97** [spyder-ide/spyder#7005]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Rinky\Anaconda3\lib\site-packages\spyder\widgets\explorer.py", line 1131, in chdir
    os.chdir(directory)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\Guest'

Traceback (most recent call last):
  File "C:\Users\Rinky\Anaconda3\lib\site-packages\spyder\widgets\explorer.py", line 1096, in go_to_previous_directory
    self.chdir(browsing_history=True)
  File "C:\Users\Rinky\Anaconda3\lib\site-packages\spyder\widgets\explorer.py", line 1134, in chdir
    except PermissionError:
UnboundLocalError: local variable 'PermissionError' referenced before assignment

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.6.1
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.25.2 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.6.4 (OK)

```

**Comment by CAM-Gerlach** [comment]
Thanks for reporting. This is a probable duplicate or near-duplicate of #5183 , which was fixed in Spyder 3.2.4. As your version of Spyder (and Python) is a year or more out of date (the current version is 3.2.8), please update with ``conda update spyder`` from the Anaconda prompt and repeat the steps you took to replicate the problem, and if reoccurs, post them here. Otherwise, this issue will be closed after one week (7 days).

**Comment by CAM-Gerlach** [comment]
Closed due to lack of response, as an assumed duplicate. If you can provide more information, please re-open and do so. Thanks.
